package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "swagger-test-templates",
-  "version": "1.4.2",
+  "version": "1.5.1",
```

Comment:
Can you please revert the version change? I will do that when releasing this post-merge.

Reply:
Done.